velox/common/memory/HashStringAllocator.cpp

```diff
@@ -56,6 +56,31 @@ void markAsFree(HashStringAllocator::Header* FOLLY_NONNULL header) {
 }
 } // namespace
 
+std::string HashStringAllocator::Header::toString() {
+  std::ostringstream out;
+  if (isFree()) {
+    out << "free: ";
```

Comment:
I would prefer `"free: true, "` to keep the string look flat, but really not strongly hold to that